addons/headless/functions/fnc_handleDisconnect.sqf

```diff
@@ -18,7 +18,21 @@
 params ["_object"];
 
 // Exit if not HC
-if !(_object in GVAR(headlessClients)) exitWith {};
+if !(_object in GVAR(headlessClients)) exitWith {
+    // End mission when no players present
+    if (GVAR(EndMission) && {!GVAR(endMissionCheckDelayed)}) then {
+        // Delay check until 2.5 minutes into the mission - wait for allPlayers to sync
+        if (CBA_missionTime < 150) then {
+            GVAR(endMissionCheckDelayed) = true;
+            [{
+                call FUNC(endMissionNoPlayers);
+            }, [], 150 - CBA_missionTime] call CBA_fnc_waitAndExecute;
+        } else {
+            call FUNC(endMissionNoPlayers);
```

Comment:
could wait ~60 seconds before executing this. It would give someone the opportunity to reconnect without having the server exit the mission right away.